pkg/customers/address.go

```diff
@@ -171,3 +172,38 @@ type updateCustomerAddressRequest struct {
 	address   `json:",inline"`
 	Validated bool `json:"validated"`
 }
+
+func addressTypeToModel(v client.AddressType) (AddressType, error) {
+	v = client.AddressType(strings.ToLower(string(v)))
+	m := map[client.AddressType]AddressType{
+		client.ADDRESSTYPE_PRIMARY:   AddressType_Primary,
+		client.ADDRESSTYPE_SECONDARY: AddressType_Secondary,
+	}
+
+	addrType, ok := m[v]
+	if !ok {
+		return 0, fmt.Errorf("unknown addresss type: '%s'", v)
+	}
+	return addrType, nil
+}
+
+type AddressType int
+
+const (
+	AddressType_Primary   AddressType = 1
+	AddressType_Secondary AddressType = 2
+)
+
+func (a AddressType) Common() client.AddressType {
```

Comment:
We could also name this to `ToClient()`, `ToView()`, `Transform()`or `Convert()`

Chime in!

Reply:
I think `ToClient()` makes sense, or `AsClientAddressType()` or something like that

Reply:
I like `ToClient()` too